JavaFolder/code/src/main/java/xyz/krakenkat/app/interview/SubArrayWithGivenSum.java

```diff
@@ -0,0 +1,44 @@
+package xyz.krakenkat.app.interview;
+
+public class SubArrayWithGivenSum {
+
+	public static void main(String[] args) {
```

Comment:
Remove the main method. Please add the required tests for your code.

Reply:
Sure!